datasets/irish/metadata.yaml

```
@@ -1,24 +1,40 @@
 #Generated automatically by pmlb/write_metadata.py
+#Reviewed by Daniel Goldberg
 dataset: irish
-description: None yet. See our contributing guide to help us add one.
-source: None yet. See our contributing guide to help us add one.
-publication: None yet. See our contributing guide to help us add one.
+description: Educational transitions of Irish schoolchildren, aged 11 in 1967.
+source: https://www.openml.org/d/451
+publication: Monitoring the Learning Outcomes of Education Systems by Vincent Greaney and Thomas Kelleghan
 task: classification
 target:
   type: binary
-  description: None yet. See our contributing guide to help us add one.
-  code: None yet. See our contributing guide to help us add one.
+  description: Leaving certificate taken
+  code: 'not_taken'=1, 'taken'=2
 features: # list of features in the dataset
   - name: Sex
     type: binary
-    description: null # optional but recommended, what the feature measures/indicates, unit
-    code: null # optional, coding information, e.g., Control = 0, Case = 1
+    description: Sex of child # optional but recommended, what the feature measures/indicates, unit
+    code: 'male'=1, 'female'=0 # optional, coding information, e.g., Control = 0, Case = 1
     transform: ~ # optional, any transformation performed on the feature, e.g., log scaled
   - name: DVRT
-    type: continuous
+    type: ordinal
+    description: Drumcondra Verbal Reasoning Test score
   - name: Educational_level
     type: discrete
```

I think, given the automatic coding, we can leave this as nominal as well.

```suggestion
    type: nominal
```

---

Sounds good